gui/app/tests/test_napari_application.py

```diff
@@ -1,13 +1,10 @@
-import pytest
 import sys
 from PyQt5.QtWidgets import QApplication
 
 from napari_application import NapariApplication
 
 
-class NapariApplicationTest(object):
-
+class TestNapariApplicationTest:
```

Comment:
Why is this written in UnitTest style instead of (imho much simpler) pytest?

```python
def test_application_type():
    app = NapariApplication(sys.argv)
    assert isinstance(app, QApplication)
```
?

Reply:
You're right, I didn't put much thought into this while I was refactoring the tests. I'll fix this soon.